src/main/java/org/openhab/binding/zwave/handler/ZWaveControllerHandler.java

```diff
@@ -69,6 +69,8 @@ public abstract class ZWaveControllerHandler extends BaseBridgeHandler implement
     private Integer secureInclusionMode;
     private Integer healTime;
     private Integer wakeupDefaultPeriod;
+    private Integer maxAwakePeriod;
+    private Integer maxAwakePeriodUpdate;
```

Comment:
This doesn't need to be defined here. From what I see, this is only used in one place so it can be a local variable rather than a field in the class.

Reply:
Without researching local and field, I moved this to the only place it is used, next to another variable that is only used there. Hopefully that is correct.  If not I'll look at in the AM.

Not a pain, trying to learn and help the OH community at the same time.

Bob

Reply:
Thanks - I created a suggestion in another comment :)

Basically it can go right down into the little bit of code where it's actually used - and given it's only created as a variable and then passed to another method, it actually doesn't need to be created at all. Take a look at the code I referenced.

Thanks.

Reply:
Done